src/routes/product-list/ProductCard.tsx

```diff
@@ -26,7 +26,17 @@ export const ProductCard = ({ product }: Props) => {
     <Link href={`/products/${product.id}`}>
       <Stack className="product-card">
         <Stack gap={10}>
-          <Image src={image} className="product-card-image w-full" />
+          <Box
+            w="100%"
+            className="product-card-image-container aspect-ratio-square"
+          >
+            <Image
+              src={image}
+              className="product-card-image object-cover"
+              w="100%"
+              h="100%"
+            />
+          </Box>
```

Comment:
Fixes layout shifts (CLS) by reserving width and height for the product images.

Reply:
How do I verify this works? I tried removing `src` from the image and it seems the container shrinks.

![image](https://github.com/user-attachments/assets/d9f490d2-e007-40f8-8a57-77a9d33dd825)

Also, we might not need this change since we already preload all product images.

Reply:
@WiNloSt Thanks for testing this! The CSS was indeed broken, I just fixed that in https://github.com/metabase/shoppy/pull/71/commits/7ac8830987ed1c3b8d9a71c37958c5f9a04f28cc. You should be able to link the `src` to a nonexistent image or just remove the `src`, and it should show up like in this screenshot:

![CleanShot 2567-10-09 at 22 57 45@2x](https://github.com/user-attachments/assets/3d732db1-5c54-4607-9d82-a55207b69963)

It's less important now that we have the preload, but still good to have in case of a slow network connection.